README.md

```diff
@@ -1,7 +1,188 @@
-## Sagemaker Container Support
+![alt text](branding/icon/sagemaker-banner.png "Logo Title Text 1")
 
-This support code is used for making machine learning frameworks (e.g. MXNet, TensorFlow) run on Amazon SageMaker. 
+# Amazon SageMaker Containers
 
-## License
+**SageMaker Containers** is the open source library that the SageMaker team uses to create the open source containers
+that run on [Amazon SageMaker](https://aws.amazon.com/documentation/sagemaker/):
+
+- https://github.com/aws/sagemaker-tensorflow-containers
+- https://github.com/aws/sagemaker-mxnet-containers
+    
+ You can use SageMaker Containers as a **library** with helper functions for your training script or to 
+ **create** your own ML framework container compatible with SageMaker. 
+ 
+## Install
+ 
+### Installing from source
+ ```bash
+pip install -U .
+```
+
+## Getting started
+### Using it as a **library**
+```sagemaker_containers.training_environment``` contains information about the training container including: 
+hyperparameters, instance information (number of CPUs, GPUs, host name), input data information, environment
+variables, etc.
+
+Let's suppose that you want to train the Keras script below in SageMaker:
+```my_training_script.py```
+```python
+import argparse
+import os
+
+import numpy as np
+import tensorflow as tf
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--training-data-dir', type=str)
+parser.add_argument('--batch-size', type=int)
+parser.add_argument('--model-dir', type=str)
+
+args = parser.parse_args()
+
+data = np.load(os.path.join(args.training_data_dir, 'training_data.npz'))
+x_train, y_train = data['features'], keras.utils.to_categorical(data['labels'])
+
+model = tf.keras.models.Sequential()
+model.add(tf.keras.layers.Dense(64, activation='relu', input_dim=20))
+model.add(tf.keras.layers.Dropout(0.5))
+
+sgd = tf.keras.optimizers.SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True)
+model.compile(loss='categorical_crossentropy', optimizer=sgd, metrics=['accuracy'])
+
+model.fit(x_train, y_train, epochs=20, batch_size=args.batch_size)
+
+# saves the model in the end of training
+model.save(os.path.join(args.model_dir, 'saved_model.h5'))
+```
+
+To be able to train the script above in SageMaker, you need to provide the parameters ```--training-data-dir```,
+```--batch-size```, ```--model-dir```. ```training_environment``` makes this process very easy:
+
+```python
+import sagemaker_containers.training_environment as env
+...
+
+# TODO (mvsusp) - create env.training_data_dir and env.evaluation_data_dir helpers
+parser.add_argument('--training-data-dir', type=str, default=env.training_data_dir)
```

Comment:
Does this actually work with the code as is right now?

the training_environment module doesn't have training_data_dir, batch_size, etc. fields. You have to create an instance of TrainingEnvironment, right?

It's worth thinking about whether there's value in needing to create an instance of the class. The functions could all just live in the module rather than the class. Is there a strong reason one way or the other currently?